### PR TITLE
poetry: add German translation

### PR DESCRIPTION
**Pull request by micschwarz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/poetry.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# poetry
+
+> Verwalte Python-Paketen und -Abhängigkeiten.
+> Mehr Informationen: <https://python-poetry.org/docs/cli/>.
```
Comment (kbdharun) [inline pages.de/common/poetry.md:4]:
```suggestion
> Weitere Informationen: <https://python-poetry.org/docs/cli/>.
```
Translate command arguments here too.

File: pages.de/common/poetry.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# poetry
+
+> Verwalte Python-Paketen und -Abhängigkeiten.
```
Comment (pixelcmtd) [inline pages.de/common/poetry.md:3]:
```suggestion
> Verwalte Python-Pakete und -Abhängigkeiten.
```

File: pages.de/common/poetry.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+> Weitere Informationen: <https://python-poetry.org/docs/cli/>.
+
+- Erstellen eins neuen Poetry-Projekts im Verzeichnis mit dem angegebenem Namen:
```
Comment (pixelcmtd) [inline pages.de/common/poetry.md:6]: I honestly hate the phrasing of the english page, but that's a discussion to be had somewhere else.

```suggestion
- Erstelle ein neues Poetry-Projekt im Verzeichnis mit dem angegebenem Namen:
```

File: pages.de/common/poetry.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`poetry new {{projekt_name}}`
+
+- Installation einer Abhängigkeit und allen Unterabhängigkeiten:
+
+`poetry add {{abhängigkeit}}`
+
+- Installation einer Entwicklungsabhängigkeit und allen Unterabhängigkeiten:
```
Comment (pixelcmtd) [inline pages.de/common/poetry.md:14]:
```suggestion
- Installiere eine Entwicklungsabhängigkeit und alle Unterabhängigkeiten:
```

File: pages.de/common/poetry.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`poetry add --dev {{abhängigkeit}}`
+
+- Interaktives Initialisieren eines neuen Poetry-Projekts im aktuellen Verzeichnis:
+
+`poetry init`
+
+- Aktualisieren aller Abhängigkeiten und `poetry.lock` aktualisieren:
```
Comment (pixelcmtd) [inline pages.de/common/poetry.md:22]: Very loose but adequate translation 😁

```suggestion
- Aktualisiere alle Abhängigkeiten und `poetry.lock`:
```

File: pages.de/common/poetry.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`poetry update`
+
+- Einen Befehl innerhalb der virtuellen Umgebung des Projekts ausführen:
```
Comment (pixelcmtd) [inline pages.de/common/poetry.md:26]:
```suggestion
- Führe einen Befehl innerhalb der virtuellen Umgebung des Projekts aus:
```

File: pages.de/common/poetry.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`poetry run {{befehl}}`
+
+- Die Minor-Version des projects in `pyproject.toml` erhöhen:
```
Comment (pixelcmtd) [inline pages.de/common/poetry.md:30]:
```suggestion
- Erhöhe die Minor-Version des Projekts in `pyproject.toml`:
```

File: pages.de/common/poetry.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+`poetry version minor`
+
+- Alle poetry Unterbefehle auflisten:
```
Comment (pixelcmtd) [inline pages.de/common/poetry.md:34]:
```suggestion
- Liste alle poetry Unterbefehle auf:
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
Please take the 5 minutes to read the contributing guide before submitting a PR to any sizable FOSS project.

File: pages.de/common/poetry.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+# poetry
+
+> Verwalte Python-Paketen und -Abhängigkeiten.
+> Weitere Informationen: <https://python-poetry.org/docs/cli/>.
+
+- Erstellen eins neuen Poetry-Projekts im Verzeichnis mit dem angegebenem Namen:
+
+`poetry new {{projekt_name}}`
```
Comment (EmilySeville7cfg) [inline pages.de/common/poetry.md:8]:
```suggestion
`poetry new "{{projekt_name}}"`
```

Optional fix (the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/poetry.md) page can be updated): possible missing quotes.

File: pages.de/common/poetry.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+`poetry init`
+
+- Aktualisieren aller Abhängigkeiten und `poetry.lock` aktualisieren:
+
+`poetry update`
+
+- Einen Befehl innerhalb der virtuellen Umgebung des Projekts ausführen:
+
+`poetry run {{befehl}}`
```
Comment (EmilySeville7cfg) [inline pages.de/common/poetry.md:28]:
```suggestion
`poetry run "{{befehl}}"`
```

The same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory fixes found, PR is approved.
- Page should be reviewed by `poetry` users.
- Thanks for a contribution! ❤️

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**